errors/B0006.md

```diff
@@ -0,0 +1,31 @@
+# B0006
+
+A runtime warning.
+
+Bevy's renderer is designed to be used with hardware acceleration. When initializing the renderer, Bevy will print an `AdapterInfo` line. If the driver in the `AdapterInfo` is indicated to be a software renderer, then the driver does not support hardware acceleration and Bevy will most likely be slow.
+
+## Possible solutions
+
+- Update your graphics driver. Your driver could simply be outdated.
+- Try using a different backend for the `RenderPlugin`, for example the OpenGL backend:
```

Comment:
I think this shouldn't be the first option and should come with big disclaimers: that's crippling rendering in the code for the dev host. It would mean every player will use OpenGL, even those with modern hardware.

Reply:
I added a disclaimer about switching to the OpenGL backend